python/orca/dev/test/run-pytests-ray

```diff
@@ -52,37 +52,56 @@ then
    exit $exit_status_2
 fi
 
-ray stop -f
+python_version=$(python --version | awk '{print$2}')
+if [ $python_version == 3.7.10 ];then
```

Comment:
Can we check the version of 3.7 instead of 3.7.10? Probably conda won't install 3.7.10 all the time?

Reply:
since we write 3.7.10 when we create the conda environment, it is ok here